deploy/complete/helm-chart/mushop/charts/orders/values.yaml

```diff
@@ -22,5 +22,9 @@ env:
   zipkin: zipkin.jaeger.svc.cluster.local
   javaOpts: -Xms128m -Xmx256m -XX:+UseG1GC -Dlogging.level.mushop.orders=TRACE  -Djava.security.egd=file:/dev/urandom -Dspring.zipkin.enabled=false
```

Comment:
Orders still need 128mb?

Reply:
updated port to be consistent.

Reply:
updated memory as well, originally fixed only in docker compose